src/job_cache/job_cache.cpp

```diff
@@ -307,6 +308,8 @@ wcl::result<FindJobResponse, FindJobError> Cache::read_impl(const FindJobRequest
 }
 
 FindJobResponse Cache::read(const FindJobRequest &find_request) {
+  wcl::log::info("Starting cache read")();
```

Comment:
nit: for these style messages I think `Cache::read enter` and `Cache::read exit` are a tiny bit more concise and a tiny bit clearer